doozerlib/cli/images_rebase.py

```diff
@@ -69,48 +69,61 @@ def images_rebase(runtime: Runtime, version: Optional[str], release: Optional[st
     metas = runtime.ordered_image_metas()
     lstate['total'] = len(metas)
 
-    def dgr_rebase(image_meta, terminate_event):
-        try:
+    def dgr_rebase(image_meta, terminate_event, runtime):
+        with runtime.db.record('rebase', metadata=image_meta) as db_entry:
+            db_entry.update({'state': 'unstarted', 'embargoed': str(embargoed)})
             dgr = image_meta.distgit_repo()
+            db_entry.update({'upstream_repo': dgr.config.content.source.git.url or '', 'upstream_branch': dgr.config.content.source.git.branch.target or ''})
```

Comment:
`image_meta.distgit_repo()` can raise an exception if it fails to clone the distgit repo. This needs to be wrapped in a `try` block otherwise the exception clause will be captured (currently we have the logic to update record.log on excpetion). in such a case a simple record like `{'state': 'unstarted', 'embargoed': str(embargoed)}` won't help much in my opinion.